### ISSUE TITLE
Refresh normalization of Timeseries

### ISSUE THREAD

**Issue by babreu-ncsa** [c3aidti/training-space-ids#462]
### Expectation
Timeseries are useful for performant aggregation/disaggregation of numerical values with point/interval timestamps across multiple instances of a source type (e.g. average wattage of multiple light bulbs within a building).

### Observation
When the timeseries objects are created, they are not readily initialized. A metric must be first evaluated so that the normalization engine is triggered and the values are sent to the key-value database (Cassandra). With that, the granularity of the values is defined by the time interval of the **first** metrics spec. After that, aggregation is naturally possible, whereas disaggregation is lost. 

### Consequence
Subsequent calls to the database with finer granularity will return a series of step functions. See screenshots below.

### Solution 
It is possible to call the ```refreshNormalization``` method of the timeseries type to trigger the normalization engine manually whenever finer granularity is desired (see below). **However, this can be quite costly when the dataset is large.**

@hgordo you may be interested in this thread.

**Comment by babreu-ncsa** [comment]
### Zooming in

- Change the spec to a single day interval (it was month above) and evaluate metrics again:
```js
var spec = EvalMetricsSpec.make({
    ids: ["1"],
    expressions: ["AverageLatitude"],
    start: "2017-08-17",
    end: "2017-08-18",
    interval: "HOUR"
});
```
![image](https://user-images.githubusercontent.com/84105092/144070066-d40720a4-ea6f-42bc-95d7-0859d10fc33a.png)


- Now fina-grain the interval spec to **MINUTE**, and there we get the series of step functions:
```js
var spec = EvalMetricsSpec.make({
    ids: ["1"],
    expressions: ["AverageLatitude"],
    start: "2017-08-17",
    end: "2017-08-18",
    interval: "MINUTE"
});
```
![image](https://user-images.githubusercontent.com/84105092/144070325-715e0d2d-5aaa-4f12-b0f2-f0033568cb46.png)

**Comment by babreu-ncsa** [comment]
# Refresh Normalization

WIP

**Comment by babreu-ncsa** [comment]
This fix happened on PR #467.

@hgordo I hope this helps!
